alacritty_terminal/CHANGELOG.md

```diff
@@ -8,11 +8,13 @@ sections should follow the order `Added`, `Changed`, `Deprecated`, `Fixed` and
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## 0.25.0-dev
+## 0.25.1-dev
+
+## 0.25.0
 
 ### Changed
 
-- Replaced `Options::hold` with `Options::drain_on_exit` that drains, but doesn't hold, since holding can be done outside of alacritty_terminal
```

Comment:
Initially wanted to reformat this to fix the horrible formatting, but decided that the extra stuff is pretty much useless anyway, so I just got rid of it.